static/docs/commands-reference/root.md

```diff
@@ -2,12 +2,18 @@
 
 Returns relative path to project's directory.
 
-Can be used to build a path to a dependency, command, or output.
+## Synopsis
 
 ```usage
 usage: dvc root [-h] [-q] [-v]
 ```
 
+## Description
+
+While modifying code in different sub-directories, there can be cases when
```

Comment:
I'm not sure I understand this to be honest, could you elaborate please? Even better can you ask a question on our Discord chat to get more information and insights?

Reply:
Ok, I'll ask on Discord. Thanks!

Reply:
Done, PTAL!